app-template/Cargo.toml

```diff
@@ -47,7 +47,7 @@ abstract-sdk = { version = "0.19.2" }
 
 # Dependencies for interface
 abstract-interface = { version = "0.19.2", optional = true }
-cw-orch = { version = "0.16.4", optional = true }
+cw-orch = { version = "0.18.1", optional = true }
```

Comment:
We can't bump this yet because it conflicts with cw-core versions used in abstract 19.x

Reply:
Downgraded